integration/kube_integration_test.go

```diff
@@ -1242,9 +1244,14 @@ func runKubeDisconnectTest(t *testing.T, suite *KubeSuite, tc disconnectTestCase
 			tty:          true,
 			stdin:        term,
 		})
-		require.NoError(t, err)
+		require.NoError(t, tc.verifyError(err))
 	}()
 
+	require.Eventually(t, func() bool {
+		// wait for the shell prompt
+		return strings.Contains(term.AllOutput(), "#")
+	}, 5*time.Second, 10*time.Millisecond)
```

Comment:
Could you add a message to the Eventually call?

Ie: require.Eventually(t, f, waitFor, tick, "If this failed it means that $message_to_add")